ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -1093,40 +1093,69 @@ public static void rename(FileSystem fs, Path src, Path dst) throws IOException,
     }
   }
 
-  private static void moveFile(FileSystem fs, FileStatus file, Path dst) throws IOException,
+  private static void moveFileOrDir(FileSystem fs, FileStatus file, Path dst) throws IOException,
```

Comment:
Can we add tests to cover this method as well ?

Reply:
Test renameOrMove files calls this, and is the public API. Tests for that should cover cases here.